src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -716,17 +773,24 @@ def resetCounters(self, rseName=None, plineName=None):
             if rseName not in self.rseCounters:
                 self.rseCounters[rseName] = {}
             self.rseCounters[rseName]['totalNumFiles'] = 0
```

Comment:
I have the impression we could simplify this `resetCounters` method. We repeat key names many times here, increasing the chances for a mistake. Could you please check whether anything can be done to enhance it?

Reply:
Already did it in the past, and couldn't figure out a way out of this. The repetition comes from the two different levels of counters we keep:
* The service counters which report at the end of every polling cycle and are preserved in the MSUnmerged instance
* The RSE counters that we need to keep track about in the database.

This is a legacy because we did not start with the stateful design from the very beginning. I am not telling there is no space for improvement though, but this would definitely cause changes in several other places in the code too.